pack/23s.json

```diff
@@ -130,9 +130,9 @@
     "position": 7,
     "quantity": 3,
     "side_code": "runner",
-    "stripped_text": "This card is not trashed until another current is played or an agenda is scored. Whenever you steal an agenda, force the Corp to \"Trace 0 If unsuccessful, the Runner gains credits equal to the number of agenda points in both players' score areas.\"",
+    "stripped_text": "This card is not trashed until another current is played or an agenda is scored. Whenever you steal an agenda, force the Corp to \"Trace[0]. If unsuccessful, the Runner gains credits equal to the number of agenda points in both players' score areas.\"",
```

Comment:
In this stripped text the trace is formatted with square brackets.

Reply:
yeah, that's intentional.  It should be the case throughout.  this enables x:"trace[3]" searches to do the right thing.